op-e2e/actions/l2_batcher_test.go

```diff
@@ -428,7 +428,7 @@ func TestBigL2Txs(gt *testing.T) {
 	}
 
 	// build many L2 blocks filled to the brim with large txs of random data
-	for i := 0; i < 100; i++ {
+	for i := 0; i < 40; i++ {
```

Comment:
@protolambda I reduced the amount of blocks here b/c this test was slow + resource intensive